Add community page with avatars of all people (#171)
open-life-science/open-life-science.github.io: community.md

---
layout: page
title: Community
---

> Thank You! to the {{ site.data.people | size }} awesome persons who participate(d) to Open Life Science! 

Reviewer: ```suggestion
Thank You! to the {{ site.data.people | size }} awesome people who participate in or contribute to Open Life Science! 
```
